docker-compose.yml

```diff
@@ -38,18 +38,21 @@ services:
     ports:
       - "80:8980"
     depends_on:
-      - app
+      - essi_app
       - worker
 
-  app:
+  essi_app:
     << : *default-app
     command: bash -l -c "rake db:migrate db:seed && yarn install && bundle exec puma -b tcp://0.0.0.0:3000"
     environment:
       VIRTUAL_HOST: essi.docker
       VIRTUAL_PORT: 3000
       IN_DOCKER: 'true'
+      HUB_URL: http://chrome:4444/wd/hub # <-- Add the HUB_URL environment variable
```

Comment:
I'm not sure this code comment adds info. it kinda says "this thing that is right here is here"